api/main_docs.go

```diff
@@ -81,6 +81,18 @@ type requestTypeListingWrapper struct {
 //   404: errorResponse
 //   500: errorResponse
 
+// swagger:route PATCH /request-types/{name} request-types updateRequestType
+//
+// Update a Request Type
+//
+// This endpoint updates an existing request type if a request type if it exists. The name cannot be changed, but
```

Comment:
Not sure, but maybe this was the intended doc?
```suggestion
// This endpoint updates an existing request type if a request type of it exists. The name cannot be changed, but
```

Reply:
Ah, that wasn't quite what I meant, but it definitely was a tpyo. Thanks for catching it!